app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
```

Comment:
We don't format parameters in this way. The best approach is: parameters start from the next line and 1 line - 1 parameter